src/features/simd.md

```diff
@@ -142,8 +142,14 @@ Another interesting set of demos that makes use of SIMD for smooth experience, c
   <figcaption>Emoji replacement</figcaption>
 </figure>
 
+## SIMD Origin Trial
+
+The WebAssembly SIMD origin trial is available for experimentation in Chrome versions 84-86. Origin trials allow developers to experiment with a feature, and provide valuable feedback. Once an origin trial token has been registered, the trial users are opted into the feature for the duration of the trial period without having to update Chrome flags. 
+
+To try this out, read the [origin trial developer guide](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md), and [register for an origin trial token](https://developers.chrome.com/origintrials/#/view_trial/-4708513410415853567). More information about origin trials can be found in the [FAQ](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#faq), please file a [bug](https://bugs.chromium.org/p/v8/issues/entry) if something isn't working as you expect. The origin trial is compatible with emscripten versions 1.39.15 onwards.
+
+Ongoing experimental support is available on a recent Chrome Canary as detailed [above](#using-webassembly-simd), with the use of latest-upstream Emscripten toolchain.
+ 
 ## Future work
 
 The current SIMD proposal is in Phase 2, so the future work here is to push the proposal forward in the standardization process. Fixed width SIMD gives significant performance gains over scalar, but it doesn’t effectively leverage wider width vector operations that are available in modern hardware. As the current proposal moves forward, some future facing work here is to determine the feasibility of extending the proposal with longer width operations.
-
-To try out current experimental support, use the latest-upstream Emscripten toolchain, and a recent Chrome Canary as detailed [above](#using-webassembly-simd). Please note that as the support is experimental, we are actively working on feature completion and performance so some breakage, or performance inconsistencies are possible.
```

Comment:
Do we not want people to use this flag for experimentation anymore?

Reply:
I've moved an abridged version of this text to be with the origin trial text, see the last sentence of that section.

Reply:
Ah cool, thanks.